Connected System Template (CST) Examples/Google Drive (OAuth,  Sending Files and Multiple Operations)/src/main/java/com/mycorp/helloworld/templates/utils/DiagnosticsFactory.java

```diff
@@ -9,19 +9,16 @@
 import com.appiancorp.connectedsystems.simplified.sdk.configuration.SimpleConfiguration;
 import com.google.api.services.drive.model.File;
 
+/**
+ * This class is an helper class that creates common fields of diagnostics objects for both Send File
```

Comment:
We need to explain what a diagnostic is

Reply:
Also get rid of this class

Reply:
@sam-a-sloate why are we killing this class? I'm not necessarily opposed but want to hear your reasoning.

Reply:
I think what Sam meant is get rid of "This class" in the comment